packages/integration-test-app/windows/integrationtest/TestHostHarness.cpp

```diff
@@ -54,8 +50,8 @@ void TestHostHarness::SetRootView(ReactRootView &&rootView) noexcept {
 void TestHostHarness::OnInstanceLoaded(const InstanceLoadedEventArgs &args) noexcept {
   VerifyElseCrash(m_dispatcher.HasThreadAccess());
 
-  if (!m_commandListener->IsListening()) {
-    StartListening();
+  if (!m_serverListenCoro) {
```

Comment:
>m_serverListenCoro [](start = 7, length = 18)

Just curious since StartListening is inside this class, why not do the noop case inside and keep it a 'void' method?